### PR TITLE
Add Alert Message for Unsigned Rec/IM Waiver

### PR DESCRIPTION
**Pull request by bennettforkner**
This PR will add a red alert message to the RecIM homepage when a participant must sign the waiver:
image
image

### CODE REVIEW

File: src/views/RecIM/views/Home/index.jsx [hunk 1]
```diff
@@ -333,15 +334,38 @@ const Home = () => {
...
-    return loading ? <GordonLoader /> : <GordonUnauthenticated feature={'the Rec-IM page'} />;
-  } else {
-    return (
-      <>
-        <Header>{headerContents}</Header>
-        {loading ? (
-          <GordonLoader />
-        ) : (
+  let headerAlert = (
+    /*participant?.Status === 'Pending' &&*/ <Alert
```
Comment (EjPlatzer) [inline src/views/RecIM/views/Home/index.jsx:338]: Looks like you commented this out for testing? This should be restored before we merge.

**Review by EjPlatzer — CHANGES_REQUESTED** [review]

**Comment by amos-cha** [comment]
I like it, but I think if we choose a warning color that would be better. The whole point is that you can hypothetically just view and watch Rec-IM as a spectator indefinitely. The error color makes it feel like you can't do anything unless you proceed.

File: src/views/RecIM/views/Home/index.jsx (hunk 1 above)
Comment (bennettforkner) [reply src/views/RecIM/views/Home/index.jsx:338]: Yea noticed that :\

Comment (bennettforkner) [inline src/views/RecIM/views/Home/index.jsx:338]:
```suggestion
    participant?.Status === 'Pending' && <Alert
```

**Comment by bennettforkner** [comment]
> I like it, but I think if we choose a warning color that would be better. The whole point is that you can hypothetically just view and watch Rec-IM as a spectator indefinitely. The error color makes it feel like you can't do anything unless you proceed.

Greg is hoping to have all students submit the waiver in this way in the long term, no matter if they are participating in Rec/IM or not. Given that, they should never see the red unless they need take some action. I considered the warning, but ended up deciding to use the alert colors to stand out as an issue.

**Comment by EjPlatzer** [comment]
> I like it, but I think if we choose a warning color that would be better. The whole point is that you can hypothetically just view and watch Rec-IM as a spectator indefinitely. The error color makes it feel like you can't do anything unless you proceed.

That's a good point. @bennettforkner what do you think about using a different color (warning or possibly even info?) instead so that people don't feel pressured to sign the waiver if they aren't planning on participating in RecIM?

**Comment by bennettforkner** [comment]
> > I like it, but I think if we choose a warning color that would be better. The whole point is that you can hypothetically just view and watch Rec-IM as a spectator indefinitely. The error color makes it feel like you can't do anything unless you proceed.
> 
> Greg is hoping to have all students submit the waiver in this way in the long term, no matter if they are participating in Rec/IM or not. Given that, they should never see the red unless they need take some action. I considered the warning, but ended up deciding to use the alert colors to stand out as an issue.

You said this but liked my reply. Where do you land now?

**Pull request merged**